changelog/@unreleased/pr-4154.v2.yml

```diff
@@ -0,0 +1,8 @@
+type: improvement
+improvement:
+  description: We no longer run Background Sweep by default if Targeted Sweep is fully
```

Comment:
try opening autorelease but on your branch to see whether or not it renders correctly

Reply:
Yep, confirmed it works on branch `0000000aaaaaaa`